src/ScheduledTask/Webhook/AuthorisationWebhookHandler.php

```diff
@@ -97,10 +106,13 @@ private function handleSuccessfulNotification(
         OrderTransactionEntity $orderTransaction,
         NotificationEntity $notification,
         Context $context
-    ) {
+    ): void {
```

Comment:
Could you please add the return and property types to all of the methods in order to be consistent?